doc/modules/calibration.rst

```diff
@@ -205,3 +205,104 @@ a similar decrease in log-loss.
     .. [5] On the combination of forecast probabilities for
            consecutive precipitation periods. Wea. Forecasting, 5, 640–650.,
            Wilks, D. S., 1990a
+
+
+Decision Threshold calibration
+==============================
+
+Often Machine Learning classifiers base their predictions on real-valued decision
+functions or probability estimates that carry the inherited biases of their models.
+Additionally when using a machine learning model the evaluation criteria can differ
+from the optimisation objectives used by the model during training.
+
+When predicting between two classes it is commonly advised that an appropriate
+decision threshold is estimated based on some cutoff criteria rather than arbitrarily
+using the midpoint of the space of possible values. Estimating a decision threshold
+for a specific use case can help to increase the overall accuracy of the model and
+provide better handling for sensitive classes.
+
+.. currentmodule:: sklearn.calibration
+
+For example the :class:`LogisticRegression` classifier is predicting the class
+for which the :func:`decision_function` returns the highest value. For a binary
+classification task that sets decision threshold to ``0``.
+
+:class:`CutoffClassifier` can be used as a wrapper around a model for binary
+classification to help obtain a more appropriate decision threshold and use it
+for predicting new samples.
+
+Usage
+-----
+
+To use the :class:`CutoffClassifier` you need to provide an estimator that has
+a ``decision_function`` or a ``predict_proba`` method. The ``scoring`` parameter
+controls whether the first will be preferred over the second if both are available.
+
+The wrapped estimator can be pre-trained, in which case ``cv='prefit'``, or not. If
+the classifier is not trained then a cross-validation loop specified by the parameter
+``cv`` can be used to obtain a decision threshold by averaging all decision thresholds
+calculated on the hold-out parts of each cross validation iteration. Finally the model
+is trained on all the provided data. When using ``cv='prefit'`` you need to make sure
+to use a hold-out part of your data for calibration.
+
+The methods for finding appropriate decision thresholds are based either on precision
+recall estimates or true positive and true negative rates. Specifically:
+
+* ``f_beta``
+   selects a decision threshold that maximizes the f_beta score. The value of
+   beta is specified by the parameter ``beta``
```

Comment:
Should probably remind the user that beta=1 balanced precision and recall, beta>1 prefers xxx, beta <1 prefers yyy.